test/blackbox/common/BlackboxTestsNetworkConf.cpp

```diff
@@ -257,77 +257,74 @@ TEST_P(NetworkConfig, PubSubInterfaceWhitelistLocalhost)
     reader.block_for_all();
 }
 
-TEST_P(NetworkConfig, PubSubInterfaceWhitelistUnicast)
+void interface_whitelist_test(
+        const std::vector<IPFinder::info_IP>& pub_interfaces,
+        const std::vector<IPFinder::info_IP>& sub_interfaces,
+        bool interface_name = false)
 {
     PubSubReader<HelloWorldPubSubType> reader(TEST_TOPIC_NAME);
     PubSubWriter<HelloWorldPubSubType> writer(TEST_TOPIC_NAME);
 
-    std::vector<IPFinder::info_IP> interfaces;
-    use_udpv4 ? GetIP4s(interfaces) : GetIP6s(interfaces);
 
-    for (const auto& interface : interfaces)
+    std::shared_ptr<UDPTransportDescriptor> pub_upv_descriptor;
+
+    if (use_udpv4)
```

Comment:
I'm not sure with UDPv6 it will work for all cases, hope it does. If so, please update the tests descriptions as they state it's UDPv4 only.

Reply:
We would need to make these tests parametrized in order to test IPv6.